collie/interactions/dataloaders.py

```diff
@@ -184,14 +184,13 @@ class ApproximateNegativeSamplingInteractionsDataLoader(BaseInteractionsDataLoad
     A computationally more efficient ``DataLoader`` for ``Interactions`` data using approximate
     negative sampling for negative items or users.
 
-    #TODO
     This DataLoader groups ``__getitem__`` calls together into a single operation, which
     dramatically speeds up a traditional DataLoader's process of calling ``__getitem__`` one index
     at a time, then concatenating them together before returning. In an effort to batch operations
     together, all negative samples returned will be approximate, meaning this does not check if a
-    user has previously interacted with the item. With a sufficient number of interactions (1M+), we
-    have found a speed increase of 2x at the cost of a 1% reduction in MAP @ 10 performance
-    compared to ``InteractionsDataLoader``.
+    user or item has previously interacted with the item or user. With a sufficient number of
```

Comment:
[Optional for fanciness suggestion]

```suggestion
    user or item has previously interacted with the item or user, respectively. With a sufficient number of
```